presto-hive/src/test/java/com/facebook/presto/hive/TestHiveUtil.java

```diff
@@ -34,6 +40,30 @@ public void testParseHiveTimestamp()
         assertEquals(parse(time, "yyyy-MM-dd HH:mm:ss.SSSSSSSSS"), unixTime(time, 7));
     }
 
+    @Test
+    public void testGetDeserializer()
+    {
+        assertTrue(getDeserializer(thriftTestSchema()) instanceof ThriftDeserializer);
```

Comment:
`Assertions` from Airlift testing has an `assertInstanceOf` method which is preferrable as it gives a better error message failure. (There a bunch of other useful assertion methods in there, too)